readthedocs/builds/models.py

```diff
@@ -756,6 +775,20 @@ def is_stale(self):
         mins_ago = timezone.now() - datetime.timedelta(minutes=5)
         return self.state == BUILD_STATE_TRIGGERED and self.date < mins_ago
 
+    @property
+    def is_external(self):
+        return self.version.type == EXTERNAL
+
+    @property
+    def external_version_name(self):
+        if self.is_external:
+            try:
+                if self.project.remote_repository.account.provider == 'github':
+                    return GITHUB_EXTERNAL_VERSION_NAME
+            except Exception:
+                return None
```

Comment:
Shouldn't this return a generic `External Version` or something?

Reply:
Yeah. that's what we were talking about. I will add that